com.github.rajsolai.textsnatcher.yml

```diff
@@ -0,0 +1,27 @@
+app-id: com.github.rajsolai.textsnatcher
+
+runtime: org.gnome.Platform
+runtime-version: "40"
+base: io.elementary.BaseApp
+base-version: "juno-20.08"
+sdk: org.gnome.Sdk
+command: com.github.rajsolai.textsnatcher
+
+finish-args:
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--share=network"
+  - "--device=dri"
+  - "--filesystem=/home"
+  - "--talk-name=org.gnome.Shell.Screenshot"
+  - "--system-talk-name=org.freedesktop.Accounts"
+
+modules:
+  - name: com.github.rajsolai.textsnatcher
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://github.com/RajSolai/TextSnatcher/archive/refs/tags/1.0.0.tar.gz
+        sha256: 5858702455a646b35a8496a44efab7216b780cdc9a0bc89e8337991c1ac1e0f5
+  - tesseract.yml
```

Comment:
splitting this out is uneccessary and make the maintainance harder.

Reply:
Sure I will make the manifest as one :+1: